pkg/event/target/webhook.go

```diff
@@ -136,15 +137,16 @@ func (target *WebhookTarget) send(eventData event.Event) error {
 // Send - reads an event from store and sends it to webhook.
 func (target *WebhookTarget) Send(eventKey string) error {
 
-	urlStr, pErr := xnet.ParseURL(target.args.Endpoint.String())
+	u, pErr := xnet.ParseURL(target.args.Endpoint.String())
```

Comment:
Rename `u` to something `urlStr`

Reply:
I cannot have the variable name as `urlStr` because of 2 reasons.
- Its of type URL not string.
- I cannot use `url` as it clashes with url package alias.